### ISSUE TITLE
Pickling the itk module fails

### ISSUE THREAD

**Issue by jakirkham** [InsightSoftwareConsortium/ITK#1050]
### Description

Pickling of the `itk` module fails.

### Steps to Reproduce

```python
import pickle
import itk                                                           

pickle.dumps(itk)
```

### Expected behavior

Pickling succeeds.

### Actual behavior

```python
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-3-8fd95ab0c545> in <module>
----> 1 pickle.dumps(itk)

TypeError: can't pickle LazyITKModule objects
```

### Reproducibility

Every time.

### Versions

ITK 5.0.0 with some modules as 5.0.0.post1.

```
$ conda list itk
...
# Name                    Version                   Build  Channel
itk                       5.0.0                    py37_1    conda-forge
itk-core                  5.0.0.post1              pypi_0    pypi
itk-filtering             5.0.0.post1              pypi_0    pypi
itk-numerics              5.0.0.post1              pypi_0    pypi
itk-registration          5.0.0.post1              pypi_0    pypi
itk-segmentation          5.0.0.post1              pypi_0    pypi
```

### Environment

Ubuntu 16.04.5 LTS (Xenial)
Python  3.7.3

**Comment by thewtex** [comment]
xref #1051

**Comment by thewtex** [comment]
Pickling via cloudpickle addressed in #1090 . We will not be able to pickle with the `pickle` module because it does not support pickling modules.
